Silence npm logs when builds fail

When tests fail, we spit out a bunch of useless `npm` error logs that have nothing to do with the actual problem, for example:

```
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! telescope@0.0.1 stylelint: `stylelint --ignore-path .gitignore .`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the telescope@0.0.1 stylelint script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/travis/.npm/_logs/2019-11-10T04_08_59_327Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! telescope@0.0.1 postlint: `npm run stylelint`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the telescope@0.0.1 postlint script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/travis/.npm/_logs/2019-11-10T04_08_59_342Z-debug.log
npm ERR! Test failed.  See above for more details.
The command "npm test" exited with 1.
```

We can suppress this noise by adding an `.npmrc` file to our project root that contains the following line:

```
loglevel=silent
```

--- a/.npmrc
+++ b/.npmrc
@@ -1,0 +1,1 @@
+loglevel=silent

